Documentation/Storage-Configuration/ceph-teardown.md

```diff
@@ -88,11 +88,11 @@ kubectl delete -f crds.yaml
 ### Delete the data on hosts
 
 !!! attention
-    The final cleanup step requires deleting files on each host in the cluster. All files under the `dataDirHostPath` property specified in the cluster CRD will need to be deleted. Otherwise, inconsistent state will remain when a new cluster is started.
+    The final cleanup step requires deleting files on each host in the cluster. All files under `dataDirHostPath` property specified in the cluster CRD will need to be deleted. Otherwise, inconsistent state will remain when a new cluster is started.
```

Comment:
Let's add back `the` in this sentence, it is a small difference, but slightly better language.

Reply:
Fixed, thanks. English is really difficult for me :-)